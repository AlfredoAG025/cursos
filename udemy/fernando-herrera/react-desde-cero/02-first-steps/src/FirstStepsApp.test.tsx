import { render, screen } from "@testing-library/react";
import { describe, expect, test } from "vitest";
import { FirstStepsApp } from "./FirstStepsApp";

describe('FirstStepApp', () => {
    test('should match snapshot', () => {
        const { container } = render(<FirstStepsApp />);
        expect(container).toMatchSnapshot();
        screen.debug();
    });
});