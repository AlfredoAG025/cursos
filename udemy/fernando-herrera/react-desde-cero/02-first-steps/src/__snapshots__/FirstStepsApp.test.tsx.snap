// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FirstStepApp > should match snapshot 1`] = `
<div>
  <h1>
    Carrito de compras
  </h1>
  <section
    class="_item-row_a1c071"
  >
    <span
      class="_item-text_a1c071"
      style="color: red;"
    >
      Nintendo Switch 2
    </span>
    <button>
      +1
    </button>
    <span>
      1
    </span>
    <button>
      -1
    </button>
  </section>
  <section
    class="_item-row_a1c071"
  >
    <span
      class="_item-text_a1c071"
      style="color: black;"
    >
      Pro Controller
    </span>
    <button>
      +1
    </button>
    <span>
      2
    </span>
    <button>
      -1
    </button>
  </section>
  <section
    class="_item-row_a1c071"
  >
    <span
      class="_item-text_a1c071"
      style="color: black;"
    >
      Super Smash
    </span>
    <button>
      +1
    </button>
    <span>
      5
    </span>
    <button>
      -1
    </button>
  </section>
</div>
`;
